src/server/scripts/EasternKingdoms/MagistersTerrace/boss_priestess_delrissa.cpp

```diff
@@ -777,69 +742,53 @@ enum HunterEnum
     SPELL_WING_CLIP             = 44286,
     SPELL_FREEZING_TRAP         = 44136,
 
-    NPC_SLIVER                  = 24552,
-
-    EVENT_SPELL_AIMED_SHOT      = 1,
-    EVENT_SPELL_SHOOT           = 2,
-    EVENT_SPELL_CONCUSSIVE_SHOT = 3,
-    EVENT_SPELL_MULTI_SHOT      = 4,
-    EVENT_SPELL_WING_CLIP       = 5
+    NPC_SLIVER                  = 24552
 };
 
 struct boss_garaxxas : public boss_priestess_lackey_commonAI
```

Comment:
Can we not move this to SAI? Looks like it uses some weird custom AI, but shouldn't the improvements in ranged AI creatures with SAI be enough to have this working correctly?